CHANGELOG.asciidoc

```diff
@@ -34,6 +34,7 @@ Use subheadings with the "=====" level for adding notes for unreleased changes:
 [float]
 ===== Features
 * Added a configuration option to use queues in names of spring-rabbit transactions - {pull}3424[#3424]
+* Bumped base alpine docker image version - {pull}3524[#3524]
```

Comment:
Sorry is an out of date Changelog, please update and add to the unreleased section above 1.47.0